Utils/Dataflow/030_PDFAnalyzer/xmltable.py

```diff
@@ -133,7 +138,8 @@ def row_centery(row):
     return c
 
 class Table:
-    re_month = re.compile("(january|february|march|april|may|june|july|august|september|october|november|december)")
+    re_month = re.compile("(january|february|march|april|may|june|july|august\
+                          |september|october|november|december)")

```

Comment:
1. Spaces around % - i made this change following overall logic of removing spaces. Also, "..."%(...) without spaces indicates that it's a single string, just defined in such way. But i'm not adamant about this and we can discuss this point. Inline comments - my apologies, i missed that, will fix.
2. Aha. I see. Missed this as well. I'll try to update my descriptions according to this, then i'll be able to say something about 72 or 100.
3. Makes sense. I'll see what can i do about changing the commit message.
4. I can't say that i'm thrilled with all this history rewriting for the sake of perfection. Especially when it requires additional work. This does not refers to the previous point, which i approve.
5. Same as 3.

Commented debug messages in the code - messages like this one are left because they are used to debug frequently performed operations, so if i'll delete them every time then i'll most likely have to write them again just as often. If logging will be introduced and it would be possible to replace commented debug messages with it - great. But right now we have no better option, right?

^M - uh, no idea what it is. I can't see them in IDLE. This and search - will check.